### PR TITLE
Portfolio mobile

### PR DESCRIPTION
**Pull request by David-Verde**
this is my first page using HTML and CSS, I´m so proud and happy I think it is perfect

### CODE REVIEW

**Comment by David-Verde** [comment]
I´m having this issue, I use visual studio limpers to clean all errors, but when I put my project in this limpers it says Error: Process completed with exit code 1. I don't know hot to fix this erros

**Comment by David-Verde** [comment]
I know the limpers must be on a green but this error doesn't have a solution I think, I use this tool PostCSS Safe Parser to clean some part of the code, could that be the problem?

**Review by OLIPLICHE — COMMENTED** [review]
Hi <STUDENT_MENTION>, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that the Linter result is not green ✅ 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by David-Verde** [comment]
ok, could you please tell me where can I find de PR thread?

File: index.html [hunk 1]
```diff
@@ -1,51 +1,77 @@
...
-   
 
-<div id="container">
 
 
 
-<img id="mylogo" src="/Portfolio-setup-and-mobile-version-skeleton/Image/Logo david.jpg" alt="">
-<div class="Header">
+  <section class="top-nav">
+    <div>
```
Comment (zainsadaqat) [inline index.html]: Please use `<header>` tag to display the navbar.
#### Example
```
<header>
  <nav>
    Navbar Code ....
  </nav>
</header>
```

File: index.html [hunk 2]
```diff
@@ -1,51 +1,77 @@
...
+  </h3>
+  <div class="icons">
+    <img src="/Image/twitter.svg" alt="">
+    <img id="Vector" src="/Image/Vector.svg" alt="">
+    <img src="/Image/medium.svg" alt="">
+    <img src="/Image/github.svg" alt="">
+    <img src="/Image/angellist.svg" alt="">
+
+
+  </div>
```
Comment (zainsadaqat) [inline index.html]: Please use `<ul>` `<li>` tags to display list of icons and use `<a>` for links.
#### Example
```
<ul>
  <li>
    <a href="github.com">
      <img src="github.jpg" alt=""/>
    </a>
  </li>
....
....
.....
</ul>
```

File: index.html [hunk 3]
```diff
@@ -1,51 +1,77 @@
...
-some of my work and experience! If you like what you see and have a project you need 
-coded, don't hestiate to contact me. 
+  <pre id="content">
+I'm a software developer! I can help you
+build a product , feature or website Look 
+through some of my work and
+experience! If you like what you see and
+have a project you need coded, don't
+hestiate to contact me. 
 </pre>
```
Comment (zainsadaqat) [inline index.html]: Please use `<p>` tag to display the paragraph for semantic reasons.

File: index.html [hunk 4]
```diff
@@ -1,51 +1,77 @@
...
-   <li>Contact</li>
-</ul>
-</nav>
-</div>
 
-<pre id="name">I'm David Verde   
-    Glad to see you! 
+  <pre id="name">I'm David Verde! 
+Glad to see 
+you!
```
Comment (zainsadaqat) [inline index.html]: As this is the main heading for your Portfolio. this should be written in `<h1>` tag Kindly replace `<pre>` with `<h1>` and there should be only one `<h1>` tag per HTML document. 👍🏻

File: index.html [hunk 5]
```diff
@@ -1,51 +1,77 @@
...
+    <ul class="menu">
+      <li>Portfolio</li>
+      <li>About</li>
+      <li>Contact</li>
+
+
+    </ul>
+  
+  </section>
+  <h1> </h1>
```
Comment (zainsadaqat) [inline index.html]: Please remove the empty tag.

File: index.html [hunk 6]
```diff
@@ -1,51 +1,77 @@
...
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
```
Comment (zainsadaqat) [inline index.html]: Please rename it with your Project name 👍🏻

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Hello @Unyielding1 😃 😄 

You're doing a great job on your project. We're proud of you. Good job so far! 
# 👏🏻 👏🏻 👏🏻 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors

## Required Changes ♻️

- [ ] Please fix the background image as per the required figma design. As you can see
[Required Figma Design]
Screen Shot 2022-04-22 at 5 52 29 AM

- [ ] [Your Design]
Screen Shot 2022-04-22 at 5 53 04 AM

- [ ] [Required Navbar]
Screen Shot 2022-04-22 at 6 01 00 AM

[Your Navbar]
- There's no need for the color gradient in the navbar as it's not in the requirements. Please remove one of them either Logo or your name from the navbar because there should be only one logo. Kindly fix this.
Screen Shot 2022-04-22 at 6 01 34 AM

_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**